README.rst

```diff
@@ -77,86 +64,46 @@ any code that was using the old version of pymedphys. We try to be abreast of
 this by providing details of any breaking changes from one release to the next
 within the `Release Notes`_.
 
+Community
+**************
 
+There is a community over at `Discourse <https://pymedphys.discourse.group/>`_
+to help you find your feet using PyMedPhys. Please reach out over there and
+we'd love to help!
 
-Documentation
--------------
-
-For App Users
-.............
-
-Documentation of the PyMedPhys application's graphical user interface doesn't
-yet exist. Watch this space.
-
-For Python Users
-................
-
-- Absolute Beginners Tutorial **[Doesn't yet exist]**
-
-  - Start here if you are new to Python and would like an in-depth instruction on
-    how to get started.
-
-- `Quickstart How-To Guide`_
-
-  - Start here if you are fluent in Python and just want a quick instruction on how
-    to get started.
 
-- `Tutorials`_
-
-  - A range of lessons that take you through various complete projects with the
-    intent to be able to show you what can be achieved with writing PyMedPhys based
-    Python software yourself.
-  - These are aimed at you if you don't yet know what you can achieve with
-    PyMedPhys but you would like to learn.
-
-- `How-To Guides`_
-
-  - Guides and recipes for common problems and tasks. These are aimed for you if
-    you already know how to use Python and PyMedPhys and are looking for direction
-    on a specific task you are trying to solve.
-
-- `Reference`_
-
-  - Technical reference for the `library`_ (modules, functions and classes),
-    as well as the available `command line tools`_. This is where you will find
-    listed information for the exposed functionality of PyMedPhys.
-  - This is aimed at you if you know exactly the feature you would like to use, you
-    just want to see what inputs it requires and what outputs it gives.
-
-- `Background`_
-
-  - Explanation and discussion of key topics and concepts. This is aimed at you if
-    you are looking to be able to think about PyMedPhys and its implementations at
-    a higher level and understand more about them.
-
-- Glossary **[Doesn't yet exist]**
+Documentation
+=============
 
-  - List of the most important terms
+The PyMedPhys documentation is split into four categories:
 
+1. `App Users Guide`_: for those who only wish to use ready-made PyMedPhys
+   tools
 
-For Contributors
-................
+.. _`App Users Guide`: https://docs.pympedphys.com/appusers/index.rst
 
-- `Getting Started Contributing`_ **[Under construction]**
+2. `Library Users Guide`_: for those building their own Python apps who wish to
+   incorporate elements of the PyMedPhys library
 
-  - Start here for learning how to contribute to PyMedPhys.
+.. _`Library Users Guide`: https://docs.pympedphys.com/libusers/index.rst
```

Comment:
These links need to be placed below ".. END_OF_DOCS_IMPORT" down the bottom. Then internal documentation links need to be provided within `index.rst`.

That way locally installed versions of the docs will send links to the local docs, not online.